gordo_components/model/factories/lstm_autoencoder.py

```diff
@@ -41,16 +44,16 @@ def lstm_model(
         One timestep = current observation in the sample.
         Two timesteps = current observation + previous observation in the sample.
         ...
-    encoding_func: list
+    encoding_dim: tuple
+        Tuple of numbers with the number of neurons in the encoding part.
```

Comment:
Minor detail, but here the documentation is using period at the end and in the ```gordo_components/model/factories/feedforward_autoencoder.py```, the documentation for encoding/decoding does not use a period at the end. Maybe we should change to one or the other?

Reply:
Grammatically speaking, since these aren't complete sentences, there shouldn't be a period; though, I am not sure what is considered standard practice here.

Reply:
We've talked about this a bit, and I agree we haven't settled and it's a bit _lazy_ at times. However, we strive to follow the numpy style of documentation, which tends to be written in sentence format. But I think we're all guilty of slipping here and there.

Reply:
Sounds good to me :+1: I love the numpy docstring format, and if complete sentences are sanctioned by them then I think that is more then enough reason to do that.

Reply:
It doesn't seem most of these are written as complete sentences. I'll just give it a whack and see if you all agree with my rephrasing of things.

Reply:
In taking a look at things, it seems like numpy does the same as we are doing here.

[Theirs](https://github.com/numpy/numpy/blob/master/numpy/core/numeric.py#L123):
```
out : ndarray
    Array of zeros with the same shape and type as `a`.
```

Ours:
```
encoding_dim: tuple
    Tuple of numbers with the number of neurons in the encoding part.
```

So here they are using an incomplete sentence with a period. Erik likes the period as it indicated the end of the definition, which I agree is a useful thing. 

In this case, I'll leave the definitions all the same, and just add `.`s where missing in those files I've handled here in this PR.